Makefile

```diff
@@ -259,8 +263,22 @@ generate-mocks: mockery ## Generate the mocks for the project as configured in .
 clean-ui:
 	find ${UI_DIR} -type f -name extension.tar -delete
 
+.PHONY: codegen-download-openapi run-backend
```

Comment:
pls fix the `.PHONY` statement. It should only contain the name of the target. 
Ex:
`.PHONY: codegen-download-openapi`